mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/services/TiltaksgjennomforingService.kt

```diff
@@ -34,9 +40,16 @@ class TiltaksgjennomforingService(
             return Either.Left(BadRequest("Avtalens sluttdato har passert"))
         }
 
+        val previousAnsvarlig = tiltaksgjennomforingRepository.get(request.id)?.ansvarlig?.navident
```

Comment:
Dette kunne vært gjort med noe sql triggers for å sjekke når ansvarlig feltet endrer seg, i stedet for å måtte hente før man upserter, men usikker på om det er noe bedre... Synes dette er enklere men sikkert litt mindre effektivt.

Reply:
Støtter enklere og mer eksplisitt fremgangsmåte :+1:

Reply:
Pluss 1 på enkelt og lesbart